Core/src/main/java/com/plotsquared/core/command/Auto.java

```diff
@@ -142,14 +142,6 @@ public static boolean checkAllowedPlots(
                 }
             }
         }
-        int maxMerge = Permissions.hasPermissionRange(player, Permission.PERMISSION_MERGE, Settings.Limit.MAX_PLOTS);
-        if (sizeX * sizeZ > maxMerge) {
```

Comment:
We could simply check `if (sizeX * sizeZ > Math.max(1, maxMerge))` here instead of reverting

Reply:
That would have been one alternative. Given this breaks a serious feature, my aim targets reverting to status quo and redo stuff from here, rather of leaving things unfixed for longer than needed.